shared/studio/tabs/auth/state/index.tsx

```diff
@@ -392,19 +392,57 @@ export class AuthAdminState extends Model({
     }
   }
 
+  @action
+  async resetCurrentEmailProvider() {
+    this.updatingEmailProviders = true;
+
+    const conn = connCtx.get(this)!;
+
+    try {
+      await conn.execute(
+        `configure current branch reset current_email_provider_name`
+      );
+      await this.refreshConfig();
+    } finally {
+      runInAction(() => (this.updatingEmailProviders = false));
+    }
+  }
+
   @computed
-  get noEmailProviderWarning() {
-    return (
-      this.providers?.some(
-        (provider) =>
-          provider._typename === "ext::auth::MagicLinkProviderConfig" ||
-          provider._typename === "ext::auth::EmailPasswordProviderConfig" ||
-          provider._typename === "ext::auth::WebAuthnProviderConfig"
-      ) &&
-      !this.emailProviders?.some(
-        (provider) => provider.name === this.currentEmailProvider
-      )
+  get emailProviderWarnings() {
+    const passwordProvider = this.providers?.find(
+      (p) => p._typename === "ext::auth::EmailPasswordProviderConfig"
+    );
+    const webauthnProvider = this.providers?.find(
+      (p) => p._typename === "ext::auth::WebAuthnProviderConfig"
+    );
+    const magicLinkProvider = this.providers?.find(
+      (p) => p._typename === "ext::auth::MagicLinkProviderConfig"
+    );
+
+    const emailVerification =
+      passwordProvider?.require_verification ||
+      webauthnProvider?.require_verification ||
+      false;
+
+    const passwordResetWebhook = this.webhooks?.some((wh) =>
+      wh.events.includes("PasswordResetRequested")
+    );
+    const magicLinkWebhook = this.webhooks?.some((wh) =>
+      wh.events.includes("MagicLinkRequested")
     );
+
+    const smtpConfigured = this.emailProviders?.some(
+      (provider) => provider.name === this.currentEmailProvider
+    );
+
+    return {
+      verificationNoSmtp: emailVerification && !smtpConfigured,
```

Comment:
verification can be a webhook as well: `EmailVerificationRequested`

https://github.com/edgedb/edgedb/blob/82fac253798a4bdccae2ebcca9aa14e3a7d0ac4d/edb/lib/ext/auth.edgeql#L376